apps/site/lib/site_web/templates/project/show.html.eex

```diff
@@ -1,21 +1,27 @@
+<%
```

Comment:
@amaisano as long as we're moving this block around anyway, could we move it into the controller rather than have the logic in the template?